src/styles/globals.scss

```diff
@@ -26,30 +26,25 @@ a {
 
 .thumbnail {
   border-radius: 50%;
-  height: 3rem;
   width: 3rem;
+  max-width: 100%;
```

Comment:
Height attribute was causing distortion. Not having max width was causing it to break the flex layout.